spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,9 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      user = sign_in_and_2fa_user
+      loa3_sp_session
```

Comment:
What do you think about moving this spec to `spec/features/saml` and using a real SAML request as opposed to stubbing the session? This will ensure a more authentic end to end test.